src/app/containers/SocialEmbed/index.jsx

```diff
@@ -32,6 +32,7 @@ const Wrapper = styled.div`
   margin-left: auto;
   margin-bottom: ${GEL_SPACING_TRPL};
   max-width: ${MAX_WIDTH};
+  ${({ maxHeight }) => (maxHeight ? `max-height: ${maxHeight}px;` : ``)}
```

Comment:
Is there a possibility that the social embed container may actually be longer than the `maxHeight`? Or if any interactivity with the embed may expand it? Are we happy with there being a scrollbar introduced if this is the case?

Reply:
There might be a case where the embed is longer than the maxHeight but we don't have such example atm. I'm trusting that the max height value comes from Ares and it corresponds to an approximate representation of how long the embed will be. Maybe @chris-hinds will know more.